source/_components/pilight.markdown

```diff
@@ -53,6 +53,11 @@ pilight:
       - 42
 ```
 
+## {% linkable_title Known issues %}
+
+- You always have to use the correct types for service data for the `pilight.send` service, as all data is sent unchecked and unchanged to the daemon. If you choose the wrong type, your call gets rejected and all you get is a `insufficient number of arguments` in the pilight log. To retrieve the right argument type, read the [implementation of your protocol](https://github.com/pilight/pilight/tree/master/libs/pilight/protocols).
```

Comment:
The message actually reads: `Pilight send failed for _message data_`. Check [here](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/pilight.py#L95).